### PR TITLE
Suggest contact when missing renku special role

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

**Comment by burnout87** [comment]
This is what I see:

image

I guess an expandable error message box would work better

**Comment by burnout87** [comment]
Better 

image

**Comment by burnout87** [comment]
image

**Comment by volodymyrss** [comment]
> image

This is not nice. The previous one is much better, and is pretty much good.
Maybe, for the previous one, it's be good to highlight  "renku contributor". Maybe just in indented new line.

**Comment by burnout87** [comment]
> it's be good to highlight "renku contributor". Maybe just in indented new line.

image

**Comment by burnout87** [comment]
> 
> the highlight is nice! what sets it?

it's a `<b>` tag for each missing role

> 
> but, the distances between word are way too large. 

Agree, will fix it

**Comment by burnout87** [comment]
The font I was previously using was creating some issues, now it's better:

image

File: cdci_data_analysis/flask_app/app.py [hunk 1]
```diff
@@ -247,10 +247,10 @@ def push_renku_branch():
     # TODO could not think of better name
     required_roles = ['renku contributor']
     if not all(item in roles for item in required_roles):
-        lacking_roles = ", ".join(sorted(list(set(required_roles) - set(roles))))
+        lacking_roles = "\n".join(['- <b>' + r + '</b>' for r in sorted(list(set(required_roles) - set(roles)))])
```
Comment (volodymyrss) [inline cdci_data_analysis/flask_app/app.py:250]: dispatcher responses are also meant for python API, so it's not good that they have HTML.
Instead, frontend might try to render output with HTML. Replace \n with   <br>, etc.

Comment (burnout87) [reply cdci_data_analysis/flask_app/app.py:250]: Taken care of in: [https://github.com/oda-hub/mmoda-frontend-module/pull/149](https://github.com/oda-hub/mmoda-frontend-module/pull/149)

**Review by volodymyrss — APPROVED** [review]

**Pull request merged**